src/cli/injected/recorder.ts

```diff
@@ -372,7 +374,7 @@ export class Recorder {
     if (event.key === '@' && event.code === 'KeyL')
       return false;
     // Allow and ignore common used shortcut for pasting.
-    if (process.platform === 'darwin') {
+    if (this._platform === 'darwin') {
```

Comment:
this is a drive-by? I assume the process.platform was being compiled by webpack into the platform of whoever compiled the bundle. Why not use navigator.platform?

Reply:
Yeah, i would also use navigator for this to feel more webby...

Reply:
Done.